[Typescript] Property 'sendVoice' does not exist on type 'Telegram'

## Context

* Telegraf.js Version: 3.29.0
* Node.js Version: v8.11.1
* Operating System: Windows 10

## Expected Behavior

_index.d.ts_ have _sendVoice_ property.

## Current Behavior

_index.d.ts_ does not have _sendVoice_ property.

## Failure Information

`bot.telegram.sendVoice(chatId, voiceId, extra)`

```
lib/bot/assistant/index.ts:724:22 - error TS2551: Property 'sendVoice' does not exist on type 'Telegram'. Did you mean 'sendInvoice'?nvoice'?

724         bot.telegram.sendVoice(config.admin, voiceId, extraVoice)
```

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -794,6 +794,13 @@ export interface Composer<TContext extends ContextMessageUpdate> {
   hears(triggers: HearsTriggers, middleware: Middleware<TContext>, ...middlewares: Array<Middleware<TContext>>): Composer<TContext>
 
   /**
+   * Registers middleware for handling callbackQuery data with regular expressions
+   * @param triggers Triggers
+   * @param middlewares Middleware functions
+   */
+  action(triggers: HearsTriggers, middleware: Middleware<TContext>, ...middlewares: Array<Middleware<TContext>>): Composer<TContext>
+
+  /**
    * Command handling.
    * @param command Commands
    * @param middlwares Middleware functions

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -232,6 +232,14 @@ export interface ContextMessageUpdate extends Context {
    */
   replyWithVideo(video: tt.InputFile, extra?: tt.ExtraVideo): Promise<tt.MessageVideo>
 
+  /**
+   * Use this method to send audio files, if you want Telegram clients to display the file as a playable voice message. For this to work, your audio must be in an .ogg file encoded with OPUS (other formats may be sent as Audio or Document). On success, the sent Message is returned. Bots can currently send voice messages of up to 50 MB in size, this limit may be changed in the future.
+   * @param voice Audio file to send. Pass a file_id as String to send a file that exists on the Telegram servers (recommended), pass an HTTP URL as a String for Telegram to get a file from the Internet, or upload a new one using multipart/form-data
+   * @param extra Additional params to send voice
+   * @returns a Message on success
+   */
+  replyWithVoice(voice: tt.InputFile, extra?: tt.ExtraVoice): Promise<tt.MessageVoice>
+
 
   // ------------------------------------------------------------------------------------------ //
   // ------------------------------------------------------------------------------------------ //
@@ -683,6 +691,15 @@ export interface Telegram {
    * @returns a Message on success
    */
   sendVideo(chatId: number | string, video: tt.InputFile, extra?: tt.ExtraVideo): Promise<tt.MessageVideo>
+
+  /**
+   * Use this method to send audio files, if you want Telegram clients to display the file as a playable voice message. For this to work, your audio must be in an .ogg file encoded with OPUS (other formats may be sent as Audio or Document). On success, the sent Message is returned. Bots can currently send voice messages of up to 50 MB in size, this limit may be changed in the future.
+   * @param chatId Unique identifier for the target chat or username of the target channel (in the format @channelusername)
+   * @param voice Audio file to send. Pass a file_id as String to send a file that exists on the Telegram servers (recommended), pass an HTTP URL as a String for Telegram to get a file from the Internet, or upload a new one using multipart/form-data
+   * @param extra Additional params to send voice
+   * @returns a Message on success
+   */
+  sendVoice(chatId: number | string, voice: tt.InputFile, extra?: tt.ExtraVoice): Promise<tt.MessageVoice>
 
   /**
    * Use this method to specify a url and receive incoming updates via an outgoing webhook

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -31,6 +31,7 @@ export interface Context {
   editedMessage?: tt.Message
   from?: tt.User
   inlineQuery?: tt.InlineQuery
+  match?: RegExpExecArray
   me?: string
   message?: tt.IncomingMessage
   preCheckoutQuery?: tt.PreCheckoutQuery

--- a/typings/telegram-types.d.ts
+++ b/typings/telegram-types.d.ts
@@ -208,6 +208,18 @@ export interface ExtraVideo extends ExtraReplyMessage {
   // https://core.telegram.org/bots/api#sendvideo
 }
 
+export interface ExtraVoice extends ExtraReplyMessage {
+  /**
+   * Voice message caption, 0-1024 characters
+   */
+  caption?: string
+
+  /**
+   * Duration of the voice message in seconds
+   */
+  duration?: number
+}
+
 export interface IncomingMessage extends TT.Message {
   audio?: TT.Audio
   entities?: TT.MessageEntity[]
@@ -261,6 +273,10 @@ export interface MessageSticker extends TT.Message {
 
 export interface MessageVideo extends TT.Message {
   video: TT.Video
+}
+
+export interface MessageVoice extends TT.Message {
+  voice: TT.Voice
 }
 
 export interface NewInvoiceParams {

--- a/typings/test.ts
+++ b/typings/test.ts
@@ -22,6 +22,8 @@ bot.command('cat', ({ replyWithPhoto }) => replyWithPhoto(randomPhoto))
 // Look ma, reply middleware factory
 bot.command('foo', reply('http://coub.com/view/9cjmt'))
 
+bot.action('bar', reply('i was here'))
+
 bot.telegram.sendMessage(process.env.BOT_CLIENT_ID,"It's work")
 
 // Start https webhook

